perun-core/src/test/java/cz/metacentrum/perun/core/entry/ModulesUtilsEntryIntegrationTest.java

```diff
@@ -715,6 +715,15 @@ public void checkQuotasErrorInValue() throws Exception {
 		modulesUtilsBl.checkAndTransferQuotas(dataQuotasAttribute, resource, null, true);
 	}
 
+	@Test(expected=WrongAttributeValueException.class)
+	public void checkQuotasErrorInPath() throws Exception {
+		System.out.println(CLASS_NAME + "WrongAttributeValueException");
```

Comment:
I think there is supposed to be name of the test in comment.

Reply:
Fixed in the original method also.